contracts/executable/AxelarGMPExecutable.sol

```diff
@@ -40,7 +41,7 @@ abstract contract AxelarGMPExecutable is IAxelarGMPExecutable {
         string calldata sourceChain,
         string calldata sourceAddress,
         bytes calldata payload
-    ) external {
+    ) external whenNotPaused {
```

Comment:
This should be added in the app logic `_execute`

Reply:
agreed. updated.